assets/javascripts/discourse/templates/modal/community-ui-builder.hbs

```diff
@@ -0,0 +1,57 @@
+<div class="resolution-form">
+  {{#d-modal-body title=formTitle class="poll-ui-builder community-ui-builder"}}
+    <form class="poll-ui-builder-form form-horizontal" {{action "submitCommunity" on="submit"}}>
+      <section class="input-group poll-textarea">
+        <label>{{i18n "communitarian.community.ui_builder.title.label"}}</label>
+        {{text-field value=title
+          class="full-width"
+          tabindex="1"
```

Comment:
Please tell me if there was any special reason to put down the `tabindex` attribute? If only for the order of switching between interactive elements, then by default (if you do not set this attribute) the order will be the same.